az-digital/az_quickstart
# Fixes #547 Create Full Calendar Filter

This PR modifies the `/calendar` page by adding a calendar selection widget for date navigation.

## Description
This PR adds an `az_calendar_filter` plugin that integrates the jqueryUI datepicker to provider an exposed calendar filter for navigation.

**The following items remain to be done:**

- [x] Convert ported JS to ES6
- [x] Adjust CSS styles for calendar 100% width on mobile in Bootstrap 4
- [x] Add in D7 functionality that highlights days with events (e.g. run view without filter and highlight cells that have events)

## Related Issue
#547 

## How Has This Been Tested?

- Verify the calendar widget is visible at `/calendar`
- Create some events
- Verify the events are visible when the appropriate days/months are selected

## Types of changes


## Checklist:
